src/stores/WatchHistoryStore.ts

```diff
@@ -28,9 +28,11 @@ export const useWatchHistoryStore = createStore<WatchHistoryState>('WatchHistory
         .watchHistory.filter(({ playlistItem, progress }) => !!playlistItem && progress > VideoProgressMinMax.Min && progress < VideoProgressMinMax.Max)
         .map(({ playlistItem }) => playlistItem),
     } as Playlist),
-  getDictionary: () =>
+  getDictionary: (useSeries: boolean = false) =>
```

Comment:
The usage of the boolean input here is hard to follow. Since each boolean case is only used in 1 place, maybe it's better to just inline both implementations?